src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,109 @@
+package duke.command;
+
+import java.time.LocalDate;
+
+import duke.Storage;
+import duke.Utility;
+import duke.exception.DukeException;
+import duke.exception.DukeExceptionType;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+
+/**
+ * Handles adding of ToDo, Event and Deadline tasks
+ */
+public class AddCommand extends Command {
+
+    /**
+     * Constructor for AddCommand, for ToDo task
+     *
+     * @param command ToDo command
+     * @param description Task name of ToDo
+     */
+    public AddCommand(String command, String description) {
+        super.command = command;
+        super.description = description;
+        super.date = "";
+    }
+
+    /**
+     * Overloaded constructor for AddCommand, for Event and Deadline tasks
+     *
+     * @param command Event or Deadline command
+     * @param description Task name of Event or Deadline
+     */
+    public AddCommand(String command, String description, String date) {
+        super.command = command;
+        super.description = description;
+        super.date = date;
+    }
+
+    private void todoProcess(String taskName, TaskList tasks) {
```

Comment:
Since it's an add command, do you think addTodo, addDeadline and addEvent are more informative naming?

I've also noticed the same issue for other Command child classes:) but I like the way you standardised similar method names across all child classes.